src/test/java/bio/terra/service/filedata/google/firestore/FileOperationTest.java

```diff
@@ -264,6 +264,15 @@ public void fileOperationsTest() throws Exception {
     @Test
     public void arrayMultiFileLoadSuccessTest() throws Exception {
         int fileCount = 10;
+
+        // Test copying load_history data in chunks
+        ConfigModel loadHistoryChunkSize = configService.getConfig(ConfigEnum.LOAD_HISTORY_COPY_CHUNK_SIZE.name());
```

Comment:
Might be worth adding a config reset() to the @After so the configuration doesn't stay with 2.
Or mark the class with the `@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_CLASS)` annotation. See FileLoadTest.

Reply:
Thank you for catching that! I added a reset() to the @after call.